Gemfile

```diff
@@ -39,4 +43,5 @@ group :test do
   gem 'database_cleaner'
   gem 'factory_girl'
   gem 'webmock'
+  gem 'sinatra-contrib'
```

Comment:
@meatballhat i think the only thing we use from `contrib` is `sinatra/test-helpers`:

```
travis-hub $ ack sinatra/
lib/travis/hub/api.rb
4:require 'sinatra/base'

spec/spec_helper.rb
56:require 'sinatra/test_helpers'
```